pyproject.toml

```diff
@@ -16,3 +16,26 @@ profile = "black"
 line_length = 120
 force_sort_within_sections = "False"
 order_by_type = "False"
+
+[tool.vulture]
+exclude = ['pytorch_lightning/metrics']
+make_whitelist = true
+min_confidence = 95
+paths = ["pytorch_lightning"]
+ignore_names = [
```

Comment:
why are these ignored, it is a kind of todo to be fixed and removed from the list?

Reply:
yes, a lot of these variables were giving false positive.